renovate.json

```diff
@@ -1,38 +1,6 @@
 {
-  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "platform": "github",
-  "labels": ["renovate"],
-  "extends": [
-    "config:base",
-    ":rebaseStalePrs",
-    "docker:disableMajor"
```

Comment:
We will follow the shared config and leave the options to maintainers. At one point you will want to do major version,